src/Orc.SupportPackage.Example/SupportPackage/Providers/CustomSupportPackageContentProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="CustomSupportPackageContentProvider.cs" company="WildGums">
+//   Copyright (c) 2008 - 2015 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Orc.SupportPackage.Example.SupportPackage.Providers
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    public sealed class CustomSupportPackageContentProvider : ISupportPackageContentProvider
+    {
+        #region Methods
+        public IEnumerable<SupportPackageFileSystemArtifact> GetSupportPackageFileSystemArtifacts()
+        {
+            var demoProjectPath = @"c:\Tmp\Orc.SupportPackage.Example";
```

Comment:
Can't we use Environment.WorkingDirectory or GetType().GetAssemblyEx().GetDirectory() instead?